apps/checkout-app/graphql/client.ts

```diff
@@ -3,4 +3,9 @@ import { createClient } from "urql";
 export const client = createClient({
```

Comment:
It's extremely important this client is NOT used in browsers, I'd put it in the `backend` directory.